include/swift/AST/ASTScope.h

```diff
@@ -333,6 +342,13 @@ class ASTScopeImpl {
 public:
   /// expandScope me, sending deferred nodes to my descendants.
   /// Return the scope into which to place subsequent decls
+  ASTScopeImpl *expandAndBeCurrentDetectingRecursion(ScopeCreator &);
+
+  /// Expand or reexpand the scope if unexpanded or if not current.
+  /// There are several places in the compiler that mutate the AST after the
+  /// fact, above and beyond adding Decls to the SourceFile. These are
+  /// documented in: rdar://53018839, rdar://53027266, rdar://53027733,
+  /// rdar://53028050
```

Comment:
I prefer not to have radar links within the source code, because they're opaque outside of Apple. Can you please remove them, mirror them in JIRA with SR- references, or otherwise remove them?

Reply:
Okey-doke. Done!